examples/NetCore2/ConsoleExampleJsonConfig/Program.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using NLog;
+using NLog.Extensions.Logging;
+
+namespace ConsoleExample
+{
+    internal static class Program
+    {
+        private static void Main()
+        {
+            var logger = LogManager.GetCurrentClassLogger();
+
+            try
+            {
+                var config = new ConfigurationBuilder()
+                    .SetBasePath(System.IO.Directory.GetCurrentDirectory())
+                    .AddJsonFile("appsettings.json", optional: true, reloadOnChange: true)
+                    .Build();
+
+                LogManager.Configuration = new NLogLoggingConfiguration(config.GetSection("NLog"));
```

Comment:
@snakefoot maybe we should do this in ` loggingBuilder.AddNLog(config);`, as we have already the config? Or is there a reason why we don't do that?

Reply:
Think the only reason is that `AddNLog(config)` has just been introduced moments ago.